ecs-cli/modules/compose/ecs/service.go

```diff
@@ -65,6 +67,27 @@ func (s *Service) LoadContext() error {
 		MaximumPercent:        maxPercent,
 		MinimumHealthyPercent: minHealthyPercent,
 	}
+
+	targetGroupArn := s.Context().CLIContext.String(TargetGroupArnFlag)
+	containerName := s.Context().CLIContext.String(ContainerNameFlag)
+	containerPort, err := getInt64FromCLIContext(s.Context(), ContainerPortFlag)
+	if err != nil {
+		return err
+	}
+	loadBalancerName := s.Context().CLIContext.String(LoadBalancerNameFlag)
+
+	s.loadBalancer = &ecs.LoadBalancer{
+		ContainerName: aws.String(containerName),
+		ContainerPort: containerPort,
+	}
+
+	if targetGroupArn != "" {
```

Comment:
It would be nice to educate the customers that `loadBalancerName` and `targetGroupArn` by throwing an error when both are specified instead of picking one for them.